### PR TITLE
chore: Add table for screen_record kwarg

### PR DESCRIPTION
**Pull request by ki4070ma**
### Background

https://github.com/appium/python-client/pull/580#issuecomment-774668557

### Note
http://appium.io/docs/en/commands/device/recording-screen/start-recording-screen/
http://appium.io/docs/en/commands/device/recording-screen/stop-recording-screen/
https://github.com/appium/appium-windows-driver/blob/master/lib/commands/record-screen.js
https://github.com/appium/appium-mac2-driver/blob/master/lib/commands/record-screen.js
https://github.com/appium/appium-android-driver/blob/master/lib/commands/recordscreen.js
https://github.com/appium/appium-xcuitest-driver/blob/master/lib/commands/recordscreen.js

### Test

image

### CODE REVIEW

File: appium/webdriver/extensions/screen_record.py [hunk 1]
```diff
@@ -29,6 +29,62 @@ class ScreenRecord(webdriver.Remote):
     def start_recording_screen(self: T, **options: Any) -> Union[bytes, str]:
         """Start asynchronous screen recording process.
 
+        +--------------+-----+---------+-----+-------+
+        | Keyword Args | iOS | Android | Win | macOS |
+        +==============+=====+=========+=====+=======+
+        | remotePath   | O   | O       | O   | O     |
```
Comment (mykola-mokhnach) [inline appium/webdriver/extensions/screen_record.py:35]: some of these args belong to stop_recording method

File: appium/webdriver/extensions/screen_record.py [hunk 2]
```diff
@@ -63,41 +119,41 @@ def start_recording_screen(self: T, **options: Any) -> Union[bytes, str]:
...
                 to 'medium'.
-            videoType (str): [iOS only] The format of the screen capture to be recorded.
+            videoType (str): [iOS] The format of the screen capture to be recorded.
                 Available formats: Execute `ffmpeg -codecs` in the terminal to see the list of supported video codecs.
                 'mjpeg' by default. (Since Appium 1.10.0)
-            videoFps (int): [iOS only] The Frames Per Second rate of the recorded video. Change this value if the
+            videoFps (int): [iOS] The Frames Per Second rate of the recorded video. Change this value if the
                 resulting video is too slow or too fast. Defaults to 10. This can decrease the resulting file size.
-            videoFilters (str): [iOS, macOS only] The FFMPEG video filters to apply. These filters allow to scale,
+            videoFilters (str): [iOS, macOS] The FFMPEG video filters to apply. These filters allow to scale,
```
Comment (mykola-mokhnach) [inline appium/webdriver/extensions/screen_record.py:129]: , Windows

File: appium/webdriver/extensions/screen_record.py [hunk 3]
```diff
@@ -29,6 +29,62 @@ class ScreenRecord(webdriver.Remote):
...
+        +--------------+-----+---------+-----+-------+
+        | headers      | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | videoQuality | O   |         |     |       |
+        +--------------+-----+---------+-----+-------+
+        | videoType    | O   |         |     |       |
+        +--------------+-----+---------+-----+-------+
+        | videoFps     | O   |         |     |       |
+        +--------------+-----+---------+-----+-------+
+        | videoFilter  | O   |         | O   | O     |
```
Comment (mykola-mokhnach) [inline appium/webdriver/extensions/screen_record.py:65]: Filters

Comment (ki4070ma) [reply appium/webdriver/extensions/screen_record.py:61]: Is `videoFilter` correct? > ref. https://github.com/appium/java-client/blob/ff10d910eae1706d4e0b4888979b9e4ffc91e35d/src/main/java/io/appium/java_client/mac/Mac2StartScreenRecordingOptions.java#L146

Comment (mykola-mokhnach) [reply appium/webdriver/extensions/screen_record.py:61]: yeah, videoFilter should be the correct one. sorry for the confusion: https://github.com/appium/appium-mac2-driver/blob/ff143194b199ae08803bce7adc7a51793c810491/lib/commands/record-screen.js#L235

**Review by mykola-mokhnach — APPROVED** [review]

File: appium/webdriver/extensions/screen_record.py [hunk 4]
```diff
@@ -29,6 +29,60 @@ class ScreenRecord(webdriver.Remote):
...
+        +--------------+-----+---------+-----+-------+
+        | user         | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | password     | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | method       | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | timeLimit    | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | user         | O   | O       | O   | O     |
```
Comment (mykola-mokhnach) [inline appium/webdriver/extensions/screen_record.py:45]: user entry is duplicated

**Comment by KazuCocoa** [comment]
nice

**Pull request merged**